client/gutenberg/extensions/subscriptions/edit.jsx

```diff
@@ -0,0 +1,69 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import { TextControl, FormToggle } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import { sprintf } from '@wordpress/i18n/build/index';
+
+class SubscriptionEdit extends Component {
+	render() {
+		const { attributes, className, isSelected, instanceId, setAttributes } = this.props;
+		const {
+			subscribe_placeholder,
+			subscribe_button,
+			success_message,
+			show_subscribers_total,
+		} = attributes;
+
+		if ( isSelected ) {
+			return (
+				<div className={ className }>
+					{ __( 'Show total subscribers' ) }
+					<FormToggle
+						checked={ show_subscribers_total }
+						onChange={ () => {
```

Comment:
Let's make sure to define this as a separate method, and not have to define it with every `render()`.